components/Tags.tsx

```diff
@@ -27,25 +28,27 @@ const Tags = ({ uniqueTags }: Props) => {
       >
         ALL
       </li>
-      {uniqueTags.map((uniqueTag) => (
-        <li
-          key={uniqueTag}
-          className={selectedTags.includes(uniqueTag) ? "active" : ""}
-          onClick={() => {
-            if (selectedTags.includes(uniqueTag)) {
-              const updatedTags = selectedTags.filter(
-                (tag) => tag !== uniqueTag
-              );
-
-              dispatch(setTags(updatedTags));
-            } else {
-              dispatch(setTags([...selectedTags, uniqueTag]));
-            }
-          }}
-        >
-          {uniqueTag}
-        </li>
-      ))}
+      {uniqueTags.map((uniqueTag, index) =>
+        countOfPostsInTags[index] === 1 ? null : (
```

Comment:
1) `1` is a magic number, so you need to explain yourself by using a `const` value
2) Use the proper `filter` method for filtration not this weird rendering technique